http.go

```diff
@@ -0,0 +1,119 @@
+package goutils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/Yapo/goutils/commonErrors"
+	"github.com/Yapo/logger"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// Response is a struct to generate a response from POST/PUT requests
+type Response struct {
+	Code int
+	Body interface{}
+}
+
+// ErrorType allow have common error on diferent projects
+type ErrorType struct {
+	Code    int
+	Message string
+}
+
+// WriteHTTPResponse write to te response stream
+func WriteHTTPResponse(w http.ResponseWriter, response *Response) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(response.Code)
+	fmt.Fprintf(w, "%s", response.Body)
+}
+
+// CreateJSON convert Body to json format
+func CreateJSON(response *Response) {
+	jsonResponse, err := json.Marshal(response.Body)
+	if err != nil {
+		logger.Info("CAN'T ENCODE \"%+v\" TO JSON", response.Body)
+		response.Body = ""
+		response.Code = http.StatusInternalServerError
+		return
+	}
+	response.Body = jsonResponse
+}
+
+// SendHTTPRequest does that
+func SendHTTPRequest(url, endpoint, method, query, body string, headers map[string]string, errors ErrorType) (int, string) {
+	req, _ := http.NewRequest(method, url+endpoint+query, strings.NewReader(body))
+	for key, value := range headers {
+		req.Header.Add(key, value)
+	}
+	logger.Debug("sending HTTP request: %+v", req)
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		logger.Error("sending HTTP request: %+v", err)
+		return errors.Code, fmt.Sprintf(errors.Message, err)
+	}
+	response, _ := ioutil.ReadAll(resp.Body)
+	return resp.StatusCode, string(response)
```

Comment:
Do we need to convert response to string at this point? Probably the caller would have extra knowledge about the conversion it actually needs and it could mean a double conversion (disregard if it's not the case)